Countries_application/Countries_application/ContentView.swift

```diff
@@ -11,14 +11,65 @@ struct ContentView: View {
     var body: some View {
         
         NavigationView {
-            
-            VStack {
-                Text("TODO:- country list")
-                
+            ZStack {
+                Color("Background")
+                    .ignoresSafeArea(.all)
                 
+                VStack {
+                    
+                    List{
+                        NavigationLink(destination: TestView()){
+                            HStack{
+                            
+                                Text("🇵🇱")
+                                    .font(.system(size: 32))
+                                VStack(alignment: .leading, spacing: 1)
+                                {
+                                    Text("Polska")
+                                        .font(.system(size: 18))
+                                    Text("Warszawa")
+                                        .font(.system(size: 14))
+                                        .foregroundColor(.gray)
+                                    
+                                }
+                                //padd
+                            }
+                        }
+                        HStack{
+                            Text("🇩🇪")
+                                .font(.system(size: 32))
```

Comment:
Maybe you don't need to specify the font size directly? See font sizes already defined in SwiftUI. 

[https://developer.apple.com/design/human-interface-guidelines/foundations/typography/](https://developer.apple.com/design/human-interface-guidelines/foundations/typography/)

[https://developer.apple.com/documentation/swiftui/font](https://developer.apple.com/documentation/swiftui/font)

You can use it like that:
For example:
`.font(.headline)`